plugins/python/python.plugin.zsh

```diff
@@ -44,7 +44,7 @@ function pyuserpaths() {
 alias pygrep='grep -nr --include="*.py"'
 
 # Run proper IPython regarding current virtualenv (if any)
-alias ipython="python3 -c 'import IPython; IPython.terminal.ipapp.launch_new_instance()'"
+alias ipython='python3 -c "import IPython, sys; IPython.start_ipython(argv=sys.argv[1:])"'
```

Comment:
It seems that it's not needed to pass `argv`, it should already parse them: https://ipython.readthedocs.io/en/stable/api/generated/IPython.html#IPython.start_ipython

Reply:
Interesting 😄